sentinel-extension/sentinel-annotation-aspectj/src/test/java/com/alibaba/csp/sentinel/annotation/aspectj/integration/service/GlobalFallback.java

```diff
@@ -0,0 +1,12 @@
+package com.alibaba.csp.sentinel.annotation.aspectj.integration.service;
+
+/**
+ * @author zhaoyuguang
+ */
+
```

Comment:
Could you please add licese header(`1999-2020`) for the class and remove this unused blank line

Reply:
fixed